portal/src/components/Materials/MaterialInfo/MaterialInfo.component.js

```diff
@@ -123,5 +129,15 @@ export default {
       return false;
     }
   },
-  watch: {}
+  watch: {
+    /**
+     * If the material changes, it is checked if the material has been rated
+     */
+    material: function () {
```

Comment:
Verandert het material zonder dat er een nieuw component wordt gemount? Als dat zo is dan werkt die applaud ook niet, maar ik verwacht dan ook eigenlijk dat er een andere instantie van dit component is. Dan zou dit niet nodig moeten zijn.